### PR TITLE
Update whats-new.md

### PR DESCRIPTION
**Pull request by Swatyario**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/whats-new.md [hunk 1]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
 
 For now, it is possible to switch back to the existing EAC (often called the "classic" EAC), but at a future date, the classic EAC will be retired.
 
+## September 2020 (Ignite Release) 
+
+Here are some of the changes and new features we introduced in the modern EAC in September 2020. 
+
+### Groups – Creation and Management 
+
+The admins can now create Groups from the modern EAC. Currently, they can create and manage M365 groups, Distribution groups and Mail-enabled Security groups.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:33]:
```suggestion
The admins can now create Groups from the modern EAC. Currently, they can create and manage Microsoft 365 groups, Distribution groups, and Mail-enabled Security groups.
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 2]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
...
+
+3 types of Groups that the admins can now create and manage are:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:35]:
```suggestion
Three types of groups that the admins can now create and manage are:  
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 3]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
 
 For now, it is possible to switch back to the existing EAC (often called the "classic" EAC), but at a future date, the classic EAC will be retired.
 
+## September 2020 (Ignite Release)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:27]:
```suggestion
## September 2020 (Ignite release) 
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 4]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
 
 For now, it is possible to switch back to the existing EAC (often called the "classic" EAC), but at a future date, the classic EAC will be retired.
 
+## September 2020 (Ignite Release) 
+
+Here are some of the changes and new features we introduced in the modern EAC in September 2020. 
+
+### Groups – Creation and Management
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:31]:
```suggestion
### Groups – creation and management 
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 5]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
...
+
+Here are some of the changes and new features we introduced in the modern EAC in September 2020. 
+
+### Groups – Creation and Management 
+
+The admins can now create Groups from the modern EAC. Currently, they can create and manage M365 groups, Distribution groups and Mail-enabled Security groups.
+
+3 types of Groups that the admins can now create and manage are:  
+
+**M365 Groups** – They are the recommended Group type and they allow for effective collaboration by providing them shared email and a shared workplace.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:37]:
```suggestion
**Microsoft 365 groups** – They are the recommended group type and they allow for effective collaboration by providing shared email and a shared workplace. 
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 6]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
...
+
+### Groups – Creation and Management 
+
+The admins can now create Groups from the modern EAC. Currently, they can create and manage M365 groups, Distribution groups and Mail-enabled Security groups.
+
+3 types of Groups that the admins can now create and manage are:  
+
+**M365 Groups** – They are the recommended Group type and they allow for effective collaboration by providing them shared email and a shared workplace. 
+
+**Distribution Groups** – They are the most used Group type and they allow you to send email to all members of the list.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:39]:
```suggestion
**Distribution groups** – They are the most used group type and they allow you to send email to all members on the list. 
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 7]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
...
+
+The admins can now create Groups from the modern EAC. Currently, they can create and manage M365 groups, Distribution groups and Mail-enabled Security groups.
+
+3 types of Groups that the admins can now create and manage are:  
+
+**M365 Groups** – They are the recommended Group type and they allow for effective collaboration by providing them shared email and a shared workplace. 
+
+**Distribution Groups** – They are the most used Group type and they allow you to send email to all members of the list. 
+
+**Mail-enabled Security Groups** – They give the members access to various resources like OneDrive, SharePoint and admin roles.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:41]:
```suggestion
**Mail-enabled security groups** – they give the members access to various resources like OneDrive, SharePoint, and admin roles. 
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 8]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
...
+
+3 types of Groups that the admins can now create and manage are:  
+
+**M365 Groups** – They are the recommended Group type and they allow for effective collaboration by providing them shared email and a shared workplace. 
+
+**Distribution Groups** – They are the most used Group type and they allow you to send email to all members of the list. 
+
+**Mail-enabled Security Groups** – They give the members access to various resources like OneDrive, SharePoint and admin roles. 
+
+### Normal/Compact list view in Groups
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:43]:
```suggestion
### Normal/Compact list view in groups 
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 9]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
...
+
+**M365 Groups** – They are the recommended Group type and they allow for effective collaboration by providing them shared email and a shared workplace. 
+
+**Distribution Groups** – They are the most used Group type and they allow you to send email to all members of the list. 
+
+**Mail-enabled Security Groups** – They give the members access to various resources like OneDrive, SharePoint and admin roles. 
+
+### Normal/Compact list view in Groups 
+
+The admins can now choose to view the list of groups in normal or compact list view way. Compact list option allows the admins to view more entries per page.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:45]:
```suggestion
Admins can now choose to view the list of groups in normal or compact list view. The compact list option allows admins to view more entries per page. 
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 10]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
...
+
+**Distribution Groups** – They are the most used Group type and they allow you to send email to all members of the list. 
+
+**Mail-enabled Security Groups** – They give the members access to various resources like OneDrive, SharePoint and admin roles. 
+
+### Normal/Compact list view in Groups 
+
+The admins can now choose to view the list of groups in normal or compact list view way. Compact list option allows the admins to view more entries per page. 
+
+### Naming Policy for Groups
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:47]:
```suggestion
### Naming Policy for groups 
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 11]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
...
+
+**Mail-enabled Security Groups** – They give the members access to various resources like OneDrive, SharePoint and admin roles. 
+
+### Normal/Compact list view in Groups 
+
+The admins can now choose to view the list of groups in normal or compact list view way. Compact list option allows the admins to view more entries per page. 
+
+### Naming Policy for Groups 
+
+The admins now have command over the Group naming policy. They can now add prefixes and suffixes for the future groups that will be created. They can also block specific words from being used in group names and aliases.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:49]:
```suggestion
Admins now have command over the group naming policy. They can now add prefixes and suffixes for the future groups that will be created. They can also block specific words from being used in group names and aliases. 
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 12]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
...
+
+### Normal/Compact list view in Groups 
+
+The admins can now choose to view the list of groups in normal or compact list view way. Compact list option allows the admins to view more entries per page. 
+
+### Naming Policy for Groups 
+
+The admins now have command over the Group naming policy. They can now add prefixes and suffixes for the future groups that will be created. They can also block specific words from being used in group names and aliases. 
+
+### Upgrade Distribution Group
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:51]:
```suggestion
### Upgrade distribution group 
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 13]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
...
+
+The admins can now choose to view the list of groups in normal or compact list view way. Compact list option allows the admins to view more entries per page. 
+
+### Naming Policy for Groups 
+
+The admins now have command over the Group naming policy. They can now add prefixes and suffixes for the future groups that will be created. They can also block specific words from being used in group names and aliases. 
+
+### Upgrade Distribution Group 
+
+The admins can now upgrade the distribution groups into the recommended Microsoft 365 (previously O 365 groups) with few clicks.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:53]:
```suggestion
Admins can now upgrade the distribution groups into the recommended Microsoft 365 groups (previously Office 365 groups) with a few clicks. 
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 14]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
...
+
+The admins now have command over the Group naming policy. They can now add prefixes and suffixes for the future groups that will be created. They can also block specific words from being used in group names and aliases. 
+
+### Upgrade Distribution Group 
+
+The admins can now upgrade the distribution groups into the recommended Microsoft 365 (previously O 365 groups) with few clicks. 
+
+### Opt-in/out 
+
+The users now have a single click Opt-in toggle button available in classic and the new Exchange Admin Center. The button allows for easy navigation between two portals and now users can try and access the new portal seamlessly.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:57]:
```suggestion
The users now have a single click **Opt-in** toggle button available in the classic and the new Exchange admin center. The button allows for easy navigation between two portals and now users can access the new portal seamlessly. 
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 15]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
...
+
+### Upgrade Distribution Group 
+
+The admins can now upgrade the distribution groups into the recommended Microsoft 365 (previously O 365 groups) with few clicks. 
+
+### Opt-in/out 
+
+The users now have a single click Opt-in toggle button available in classic and the new Exchange Admin Center. The button allows for easy navigation between two portals and now users can try and access the new portal seamlessly. 
+
+### Support Assistant
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:59]:
```suggestion
### Support assistant 
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 16]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
...
+
+### Opt-in/out 
+
+The users now have a single click Opt-in toggle button available in classic and the new Exchange Admin Center. The button allows for easy navigation between two portals and now users can try and access the new portal seamlessly. 
+
+### Support Assistant 
+
+The admins can now get their queries resolved without going anywhere else. The support assistant feature allows users to resolve the queries without leaving the EAC window. In case the user is not satisfied with the provided solution, the user can even raise a ticket and register their issues. 
+
+### Educational Navigation Video
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:63]:
```suggestion
### Educational navigation video 
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 17]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
...
+
+The users now have a single click Opt-in toggle button available in classic and the new Exchange Admin Center. The button allows for easy navigation between two portals and now users can try and access the new portal seamlessly. 
+
+### Support Assistant 
+
+The admins can now get their queries resolved without going anywhere else. The support assistant feature allows users to resolve the queries without leaving the EAC window. In case the user is not satisfied with the provided solution, the user can even raise a ticket and register their issues. 
+
+### Educational Navigation Video 
+
+An educational navigation video is now available on the very first tile ‘Training and Guide’ in the new EAC portal. The video allows the users to understand the basic navigation of the new portal. You can also access the video here: https://www.microsoft.com/en-us/videoplayer/embed/RE4FqDa.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:65]:
```suggestion
An educational navigation video is now available on the very first tile "Training and Guide" in the new EAC portal. The video allows the users to understand the basic navigation of the new portal. You can also access the video [here](https://www.microsoft.com/videoplayer/embed/RE4FqDa.
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 18]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
...
+
+### Support Assistant 
+
+The admins can now get their queries resolved without going anywhere else. The support assistant feature allows users to resolve the queries without leaving the EAC window. In case the user is not satisfied with the provided solution, the user can even raise a ticket and register their issues. 
+
+### Educational Navigation Video 
+
+An educational navigation video is now available on the very first tile ‘Training and Guide’ in the new EAC portal. The video allows the users to understand the basic navigation of the new portal. You can also access the video here: https://www.microsoft.com/en-us/videoplayer/embed/RE4FqDa. 
+
+### Recipients – MS Documents Updated
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:67]:
```suggestion
### Recipients – Microsoft documents updated 
```

File: Exchange/ExchangeOnline/whats-new.md [hunk 19]
```diff
@@ -24,6 +24,50 @@ Exchange admin center now uses a new portal at [https://admin.exchange.microsoft
...
+
+The admins can now get their queries resolved without going anywhere else. The support assistant feature allows users to resolve the queries without leaving the EAC window. In case the user is not satisfied with the provided solution, the user can even raise a ticket and register their issues. 
+
+### Educational Navigation Video 
+
+An educational navigation video is now available on the very first tile ‘Training and Guide’ in the new EAC portal. The video allows the users to understand the basic navigation of the new portal. You can also access the video here: https://www.microsoft.com/en-us/videoplayer/embed/RE4FqDa. 
+
+### Recipients – MS Documents Updated 
+
+Going forward with helping the users with their queries related to new Exchange Admin Center portal, the Microsoft documentation is also complete for the high traffic pages under Recipients.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/whats-new.md:69]:
```suggestion
In moving forward to help the users with their queries related to the new Exchange admin center portal, we have also completed the Microsoft documentation for high traffic pages under "Recipients".
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by Swatyario** [comment]
Hi Johan, Thanks for suggesting these important changes. I will take care of that in my next post. Btw, I was not able to add pictures in the blog. Here is the link of the Word Doc:

https://microsoft-my.sharepoint.com/:w:/p/adbhale/ETytKyo0LKxAlYAmpUt2CR0BrenUfm3yuUd09q3TidM6Cg?e=tbhsr8

Could you please help me add the images as well at the respective places?

**Review by JohanFreelancer9 — APPROVED** [review]
Thank you, looks good. Unfortunately, I am not able to assist with linking the images. @scanum?

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by msdmaguire** [comment]
@Swatyario, apologies that this PR fell through the cracks. I am closing this because in the interim I have worked with Aditya to update the What's new article in both September and December of 2020. If there are still gaps, please open a new PR and ping me directly when it is ready and I'll push it through.

**Pull request closed without merging**